extensions/notebook/src/jupyter/jupyterSessionManager.ts

```diff
@@ -335,8 +335,12 @@ export class JupyterSession implements nb.ISession {
 			let allCode: string = '';
 			for (let i = 0; i < Object.keys(process.env).length; i++) {
 				let key = Object.keys(process.env)[i];
-				// Jupyter doesn't seem to alow for setting multiple variables at once, so doing it with multiple commands
-				allCode += `%set_env ${key}=${process.env[key]}${EOL}`;
+				if (key.toLowerCase() === 'path' && this._pythonEnvVarPath) {
+					allCode += `%set_env ${key}=${this._pythonEnvVarPath}${EOL}`;
+				} else {
```

Comment:
Is it intended that you're not adding on the existing env vars?

Reply:
pythonEnvVarPath already does the prepending to the PATH. thanks